tests/models/auto/test_modeling_auto.py

```diff
@@ -372,3 +375,14 @@ def test_cached_model_has_minimum_calls_to_head(self):
             self.assertEqual(counter.get_request_count, 0)
             self.assertEqual(counter.head_request_count, 1)
             self.assertEqual(counter.other_request_count, 0)
+
+    def test_attr_not_existing(self):
+
+        from transformers.models.auto.auto_factory import _LazyAutoMapping
+
+        _CONFIG_MAPPING_NAMES = OrderedDict([("bert", "BertConfig")])
+        _MODEL_MAPPING_NAMES = OrderedDict([("bert", "GhostModel")])
+        _MODEL_MAPPING = _LazyAutoMapping(_CONFIG_MAPPING_NAMES, _MODEL_MAPPING_NAMES)
+
+        with pytest.raises(ValueError, match=r"Could not find GhostModel neither in .* nor in .*!"):
+            _MODEL_MAPPING[BertConfig]
```

Comment:
Are you super happy now ..?

Reply:
Could you also test with a bert ->BertModel (will succeed) and a bert->GPT2Model (will also succeed thanks to the default to transformers).

Reply:
Now I'm super happy :-)